Docs/contributing.md

```diff
@@ -1,39 +1,35 @@
+Once you get the tools you need to work with Azure functions, you can run the apps locally.
 
-You can either get the tools [integrated with Visual Studio](https://blogs.msdn.microsoft.com/webdev/2017/05/10/azure-function-tools-for-visual-studio-2017/) and use `F5`
-or you can [get the CLI](https://github.com/Azure/azure-functions-cli) standalone and use `func run Imgbot.Function`.
-If you are using Visual Studio for Mac there is [built-in support](https://docs.microsoft.com/en-us/visualstudio/mac/azure-functions) for Azure functions.
+You can either get the tools [integrated with Visual Studio](https://blogs.msdn.microsoft.com/webdev/2017/05/10/azure-function-tools-for-visual-studio-2017/) and use `F5` or you can [get the CLI](https://github.com/Azure/azure-functions-cli) standalone and use `func run Imgbot.Function`.
+If you are using Visual Studio for Mac, there is [built-in support](https://docs.microsoft.com/en-us/visualstudio/mac/azure-functions) for Azure functions.
 
-We also have support for running with VS Code. You will still need to get the CLI as mentioned above and the C# extension for VS Code in order to compile and get intellisense.
-Each function has a task you can execute that will clean + build + run the process. To start one open the prompt with `cmd/ctrl + shift + p` and select `Run task`. From there you will see all the tasks checked into `.vscode/tasks.json`. Choose a function to run such as `Run CompressImagesFunction` and it will build and start up. To attach to this process choose the `Debug a function` configuration from the debug tab to see the running processes. Type `func` into the picker to see your running function and select it. It's a two-step process, the debugger and the function process. When you kill the debugger, the process will still be running. You can kill the function host by bringing up the prompt again with `cmd/ctrl + shift + p` and select `Kill the active terminal instance`.
+We also have support for running with VS Code. You will still need to get the CLI as mentioned above and the C# extension for VS Code to compile and get intellisense.
+Each function has a task you can execute that will clean + build + run the process. To start one, open the prompt with `cmd/ctrl + shift + p` and select `Run task`. From there you will see all the tasks checked into `.vscode/tasks.json`. Choose a function to run, such as `Run CompressImagesFunction`, and it will build and start up. To attach to this process choose the `Debug a function` configuration from the debug tab to see the running processes. Type `func` into the picker to see your running function and select it. It's a two-step process, the debugger and the function process. When you kill the debugger, the process will still be running. You can kill the function host by bringing up the prompt again with `cmd/ctrl + shift + p` and select `Kill the active terminal instance`.
```

Comment:
shorter?

Reply:
```suggestion
We support running VS Code. You will still need to get the CLI (mentioned above) and the VS Code C# extension to compile and get IntelliSense.
Each function has an executable task that will clean + build + run the process. To start a task, open the prompt with `cmd/ctrl + shift + p` and select `Run task`. You will see all tasks checked into `.vscode/tasks.json`. Choose a function to run, such as `Run CompressImagesFunction`, and it will build and start-up. To attach to this process, choose the `Debug a function` configuration from the debug tab to see the running processes. Type `func` into the picker to see your running function and select it. The two-step process runs the debugger and the function process. When you kill the debugger, the function process will still be running. You can kill the function host by bringing up the prompt again with `cmd/ctrl + shift + p` and select `Kill the active terminal instance`.
```